spinalcordtoolbox/scripts/sct_compute_compression.py

```diff
@@ -73,25 +73,38 @@ def get_parser():
         """),  # noqa: E501 (line too long)
     )
     mandatory.add_argument(
+        '-l',
+        metavar=Metavar.file,
+        help=textwrap.dedent("""
+            A NIfTI file that includes either i) compression labels or ii) lesion mask.
+              i) compression labels = labels at the compression sites. Each compression site is denoted by a single voxel of value `1`. Example: `sub-001_T2w_compression_labels.nii.gz`
+              ii) lesion mask = binary mask of the lesion. Currently only a single lesion is supported. Example: `sub-001_T2w_lesion.nii.gz`
```

Comment:
It might be worth adding a note here that if a lesion mask is supplied, then `-mode lesion` also has to be set, or else the lesion's voxels will be treated like a collection of compression labels.

I think there are possibly other ideas to make the functionality more discoverable or automatic for the user, but this is the simplest approach and shouldn't hold up the PR much longer. (Sorry about the last minute feedback!)

Reply:
Excellent point, thank you! Added in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4862/commits/60b0130d630f262bf2ee0cbb6dc4ec7a4ab295cf.